### DISCUSSION TITLE
how can i use letsencrypt to enable ssl with uws ?

### DISCUSSION THREAD

**Discussion by skinkairewalker** [uNetworking/uWebSockets#1648] [General]
Hello everyone!
I'm trying to learn more about uwebsockets, and I'd like to know if it's possible to use Let's Encrypt SSL in uwebsockets..

**Comment by e3dio** [comment]
Use the generated `privkey.pem` and `fullchain.pem` files like this `SSLApp({ key_file_name: 'path/to/privkey.pem', cert_file_name: 'path/to/fullchain.pem' })`
